extractor/filesystem/os/kernel/module/metadata.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package module
+
+// Metadata holds parsing information for an module package.
+type Metadata struct {
+	PackageName                    string
+	PackageVersion                 string
+	PackageVermagic                string
+	PackageSourceVersionIdentifier string
+	OSID                           string
+	OSVersionCodename              string
+	OSVersionID                    string
+	PackageDependencies            string
```

Comment:
This field might make the output size get big so I'd rather remove it from the metadata for now.